### PR TITLE
grafana: New grafana dashboard for mgmt

### PR DESCRIPTION
**Pull request by roidelapluie**
Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

## Tips:

* commit message titles must be in the form:
```topic: Capitalized message with no trailing period```
or:
```topic, topic2: Capitalized message with no trailing period```

* golang code must be formatted according to the standard, please run:
```
make gofmt		# formats the entire project correctly
```
or format a single golang file correctly:
```
gofmt -w yourcode.go
```

* please rebase your patch against current git master:
```
git checkout master
git pull origin master
git checkout your-feature
git rebase master
git push your-remote your-feature
hub pull-request	# or submit with the github web ui
```

* after a patch review, please ping @purpleidea so we know to re-review:
```
# make changes based on reviews...
git add -p		# add new changes
git commit --amend	# combine with existing commit
git push your-remote your-feature -f
# now ping @purpleidea in the github PR since it doesn't notify us automatically
```

### CODE REVIEW

**Comment by jonathangold** [comment]
Cool! (｡◕‿◕｡)

File: .gitmodules [hunk 1]
```diff
@@ -22,3 +22,6 @@
 [submodule "vendor/github.com/ugorji/go"]
 	path = vendor/github.com/ugorji/go
 	url = https://github.com/ugorji/go
+[submodule "grafana/grafonnet-lib"]
```
Comment (purpleidea) [inline .gitmodules]: Is there a reason this is vendored instead of using git master?

File: grafana/Makefile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+build:
+	jsonnet --ext-str-file logo=../art/mgmt_logo_default_tall.svg -J grafonnet-lib mgmt.jsonnet > mgmt.json
```
Comment (purpleidea) [inline grafana/Makefile]: nice :)

File: grafana/README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Grafana dashboard for mgmt
```
Comment (purpleidea) [inline grafana/README.md]: Want to add a link to this file from README.md in the docs/ section?

I think this README should me in docs/ instead. Are you okay with that?

**Pull request closed without merging**